### DISCUSSION TITLE
Option to sort chats in descending order

### DISCUSSION THREAD

**Discussion by haoblackj** [danny-avila/LibreChat#4025] [General]
Hi,there.

Currently, the chat is in the following order.

Today→yesterday→previous 7 days→previous 30 days→each month (June,July,August,etc...)

Of these, only the order of each month is ascending. In other words, the next month after the previous 30 days is not August, but the month in which I started using LibreChat.

I would like to have the option to select the order of the chats (sorry if this is already there!). Thank you for your help.

**Comment by haoblackj** [comment]
I'm not from an English-speaking country, so I'm sorry if my English is difficult to understand.
Also, I'm not involved in software development on a regular basis, so I might make mistakes in the creation of Discussions or the content of my contributions.
If there is any information missing in the above post, I would appreciate it if you could let me know in your reply.

**Reply by danny-avila** [reply]
I believe this issue was fixed already on the latest update, sorry I missed your initial post

**Reply by haoblackj** [reply]
I updated it and now it appears as I expected! Thanks for fixing it!
